### PR TITLE
Master sale layout newapi rop

### PR DESCRIPTION
**Pull request by rop-odoo**
Task: https://www.odoo.com/web?#id=13369&view_type=form&model=project.task&menu_id=3942&action=327
Pad: https://pad.odoo.com/p/r.x92W9sX4VyYKX9B9

### CODE REVIEW

File: addons/sale_layout/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+from openerp import api, models
+from openerp.addons.sale_layout.models.sale_layout import grouplines
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    @api.model
+    def sale_layout_lines(self):
```
Comment (atp-odoo) [inline addons/sale_layout/models/account_invoice.py]: use @api.multi

File: addons/sale_layout/models/account_invoice.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        """
+        Returns invoice lines from a specified invoice ordered by
+        sale_layout_category sequence. Used in sale_layout module.
+        """
+        # We chose to group first by category model and, if not present, by invoice name
+        sortkey = lambda x: x.sale_layout_cat_id if x.sale_layout_cat_id else ''
```
Comment (atp-odoo) [inline addons/sale_layout/models/account_invoice.py]: no need to check if and else condition

File: addons/sale_layout/models/sale_order.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+from openerp import api, models
+from openerp.addons.sale_layout.models.sale_layout import grouplines
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    @api.model
+    def sale_layout_lines(self):
```
Comment (atp-odoo) [inline addons/sale_layout/models/sale_order.py]: same here as above use @api.multi

**Comment by rop-odoo** [comment]
continue on PR: https://github.com/odoo/odoo/pull/8520

**Pull request closed without merging**